bin/node/src/commands/store.rs

```diff
@@ -177,3 +177,13 @@ impl StoreCommand {
         ))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::StoreCommand;
+
+    #[test]
+    fn test_generate_genesis_account_no_panic() {
```

Comment:
I'd like to move away from prefixing every test with `test_`.

```suggestion
    fn generate_genesis_account_succeeds() {
```